site/en/blog/new-in-chrome-94/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: New in Chrome 94
+description: >
+  Chrome 94 is rolling out now! The default color space for `<canvas>`
+  elements is now formally defined as sRBG, and you can change it to Display
```

Comment:
I think this should be "sRGB", here and below.

Reply:
Whoops, correct!